spec/features/scheduling/update_proposed_publish_time_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.feature "Update proposed publish time" do
+  include ActiveSupport::Testing::TimeHelpers
+
+  around do |example|
+    travel_to(Time.zone.parse("2019-06-13")) { example.run }
+  end
+
+  scenario do
+    given_there_is_an_edition_with_a_proposed_publish_time
```

Comment:
Minor, but looks like we've now got a concise 'schedulable' term we can use here :-).